tools/cli/src/args.rs

```diff
@@ -0,0 +1,92 @@
+use clap::Parser;
+use clap::{Args, Subcommand};
+use std::path::PathBuf;
+
+#[derive(Parser)]
+#[command(version, about, long_about = None)]
+pub struct CliArgs {
+    #[arg(long, value_enum, default_value_t = TargetEnvironment::Local)]
```

Comment:
NB: we will also want people to be able to define their own endpoints, override these defaults.

At minimum, we need a new one for layer-p. But people should be able to set the config entirely via cli args / env vars / toml file. Let's discuss this here https://github.com/Lay3rLabs/avs-toolkit/issues/30

Reply:
agreed, shared my thoughts on how to go about it on that issue